Libs/MRML/Core/vtkMRMLSliceNode.cxx

```diff
@@ -751,6 +879,26 @@ void vtkMRMLSliceNode::WriteXML(ostream& of, int nIndent)
       }
     }
   of << indent << " sliceToRAS=\"" << ss.str().c_str() << "\"";
+
+  std::map<vtkStdString, vtkMatrix4x4*>::iterator it;
+  for (it = this->OrientationMatrices.begin(); it != this->OrientationMatrices.end(); ++it)
+    {
+    std::stringstream ss;
+    int j;
+    for (i=0; i<4; i++)
```

Comment:
see up. Of course the behaviour can be changed, and I can store MAtrix3x3 for the orietation and then make the copy to SliceToRAS manually element by element, instead to use DeepCopy